File: src/main/java/src/SistemaGestionEstudiante/Gestor.java
package SistemaGestionEstudiante;

public class Gestor {
    public static void main(String[] args) {
        Escuela escuela = new Escuela();

        Profesor profesor1 = new Profesor("Juan Pérez");
        Profesor profesor2 = new Profesor("María López");

        Curso curso1 = new Curso("Matemáticas", profesor1);
        Curso curso2 = new Curso("Historia", profesor2);

        escuela.agregarCurso(curso1);
        escuela.agregarCurso(curso2);

        Estudiante estudiante1 = new Estudiante("Carlos");
        Estudiante estudiante2 = new Estudiante("Ana");

        escuela.agregarEstudiante(estudiante1);
        escuela.agregarEstudiante(estudiante2);

        profesor1.asignarCalificacion(estudiante1, 55);
        profesor1.asignarCalificacion(estudiante1, 77);
        profesor2.asignarCalificacion(estudiante2, 90);
        profesor2.asignarCalificacion(estudiante2, 82);


        escuela.mostrarEstudiantes();
        escuela.mostrarCursos();
    }
}

File: src/main/kotlin/SistemaGestionEstudiante/Gestor.kt
package org.example.SistemaGestionEstudiante

fun main() {
    val escuela: Escuela = Escuela()

    val profesor1 = Profesor("Juan Pérez")
    val profesor2 = Profesor("María López")

    val curso1 = Curso("Matemáticas", profesor1)
    val curso2 = Curso("Historia", profesor2)

    escuela.agregarCurso(curso1)
    escuela.agregarCurso(curso2)

    val estudiante1 = Estudiante("Carlos")
    val estudiante2 = Estudiante("Ana")

    escuela.agregarEstudiante(estudiante1)
    escuela.agregarEstudiante(estudiante2)

    profesor1.asignarCalificacion(estudiante1, 55.0)
    profesor1.asignarCalificacion(estudiante1, 77.0)
    profesor2.asignarCalificacion(estudiante2, 90.0)
    profesor2.asignarCalificacion(estudiante2, 82.0)


    escuela.mostrarEstudiantes()
    escuela.mostrarCursos()
}